app/controllers/Application.scala

```diff
@@ -52,6 +52,16 @@ class Application @Inject()(actorSystem: ActorSystem,
     }
   }
 
+  def ingestAnalyticsEvents: Action[AnalyticsEventsIngestJson] = Action.async(validateJson[AnalyticsEventsIngestJson]) {
+    implicit request =>
+      {
+        for {
+          _ <- bool2Fox(conf.BackendAnalytics.databaseEnabled) ?~> "Database logging of events is not enabled"
+          _ <- analyticsService.ingest(request.body.events, request.body.apiKey)
+        } yield Ok
+      }
+  }
```

Comment:
Maybe extract to new AnalyticsController since we now have two routes concerned with this